apps/fortuna/src/command/run.rs

```diff
@@ -184,14 +189,15 @@ pub async fn run(opts: &RunOptions) -> Result<()> {
 
     let metrics_registry = Arc::new(RwLock::new(Registry::default()));
 
-    if let Some(keeper_private_key) = opts.load_keeper_private_key()? {
-        spawn(run_keeper(
-            chains.clone(),
-            config.clone(),
-            keeper_private_key,
-            metrics_registry.clone(),
-        ));
-    }
+    let keeper_private_key = config.keeper.private_key.load()?.ok_or(anyhow!(
```

Comment:
drive-by: disable option to run service without the keeper. I think we don't really care about the webservice only version anymore so this seems safer. (but open to debate on this)

Reply:
we deliberately did this to avoid running the keeper in multiple clusters since they would compete for fulfilling callbacks and we will have nonce issues.

Reply:
ok i'll revert this